tmk_core/common/mousekey.c

```diff
@@ -36,6 +37,7 @@ static void           mousekey_debug(void);
 static uint8_t        mousekey_accel  = 0;
 static uint8_t        mousekey_repeat = 0;
 static uint16_t       last_timer      = 0;
+static uint16_t       mouse_timer     = 0;
```

Comment:
Looks like this isn't used, unless kinetic speed is enabled. Should be wrapped in an ifdef, so that it doesn't generate errors if it's not enabled:

```suggestion
#ifdef MK_KINETIC_SPEED 
static uint16_t       mouse_timer     = 0;
#endif
```

Reply:
Done